README.md

```diff
@@ -6,7 +6,7 @@ This article shows how to setup a Docker image with the Actian OpenROAD Server o
 
 Before running the script 
 
-- Git clone will extract the files and place them in a directory (e.g. `openroad_server_docker`) which will be the base directory for the setup. `git clone https://alm.actian.com/bitbucket/scm/or/openroad_server_docker.git`
+- Git clone will extract the files and place them in a directory (e.g. `openroad_server_docker`) which will be the base directory for the setup. `git clone https://github.com/ActianCorp/OpenROAD_docker.git`
```

Comment:
The command `git clone https://github.com/ActianCorp/OpenROAD_docker.git` will create a directory called `OpenROAD_docker` rather than `openroad_server_docker`.
So either you change all the references to this directory name, or add the target directory to the `git clone` command.

Reply:
Thanks, changed the base directory name.